src/backends/tensorrt/plan_backend.h

```diff
@@ -232,8 +232,9 @@ class PlanBackend : public InferenceBackend {
     // Assume that the lifetime of the payload is extended until the completion
     // callback is called
     SyncQueue<std::tuple<
```

Comment:
This giant magic tuple is impossible to understand. Can you make it an actual struct.

Reply:
Done